magma-feedstock/recipe/meta.yaml

```diff
@@ -2,38 +2,48 @@
 # TODO: examine if it is possible to link DSO dynamically
 package:
   name: magma
-  version: 2.5.0
+  version: 2.7.0
 
 source:
-   url: http://icl.cs.utk.edu/projectsfiles/magma/downloads/magma-2.5.0.tar.gz
-   sha256: 4fd45c7e46bd9d9124253e7838bbfb9e6003c64c2c67ffcff02e6c36d2bcfa33
+   url: http://icl.cs.utk.edu/projectsfiles/magma/downloads/magma-2.7.0.tar.gz
+   sha256: fda1cbc4607e77cacd8feb1c0f633c5826ba200a018f647f1c5436975b39fd18
    patches:
-    - cmakelists_cuda90.patch   # [cudatoolkit=='9.0']
-    - cmakelists_cuda100.patch  # [cudatoolkit=='10.0']
-    - xhsgetrf_gpu.patch        # [cudatoolkit=='9.0']
     - thread_queue.patch
 
 build:
   number: 0
-  skip: True  # [win and vc<14]
-  skip: True  # [win32]
+  skip: True  # [osx or win32]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
-    - ninja  # [win]
-    - patch       # [not win]
+    - ninja       # [win]
     - m2-patch    # [win]
+    - patch       # [not win]
   host:
     - cudatoolkit {{ cudatoolkit }}*
+    - mkl
   run:
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
 
+test:
+  commands:
+    - test -f $PREFIX/include/magma.h                             # [unix]
+    - test -f $PREFIX/lib/libmagma${SHLIB_EXT}                    # [unix]
+    - test -f $PREFIX/lib/libmagma_sparse${SHLIB_EXT}             # [unix]
+    - if not exist %LIBRARY_PREFIX%\include\magma.h exit 1        # [win]
+    - if not exist %LIBRARY_PREFIX%\lib\magma.lib exit 1          # [win]
+    - if not exist %LIBRARY_PREFIX%\bin\magma.dll exit 1          # [win]
+    - if not exist %LIBRARY_PREFIX%\lib\magma_sparse.lib exit 1   # [win]
+    - if not exist %LIBRARY_PREFIX%\bin\magma_sparse.dll exit 1   # [win]
+
 about:
-  home: http://icl.cs.utk.edu/magma/software/index.html
+  home: https://icl.utk.edu/magma/index.html
   license: BSD 3-Clause
   license_family: BSD
   license_file: COPYRIGHT
   summary: Matrix Algebra on GPU and Multicore Architectures
```

Comment:
Missing a `description`. This can be used https://icl.utk.edu/projectsfiles/magma/doxygen/index.html